### DISCUSSION TITLE
Functionality to delete namespaces

### DISCUSSION THREAD

**Discussion by sero1975** [klausmeyer/docker-registry-browser#419] [Q&A]
Hi there gurus!
Is there an option to delete empty namespaces somehow?
Thanks!

**Comment by klausmeyer** [comment]
Hey,

I think this relates to #5 and #268.

Currently there is no way to delete or hide them from this application as the API provided by the registry has a few limitations in that regard.

Best would be to setup automatic [garbage collection](https://docs.docker.com/registry/garbage-collection/) in the registry or delete them manually there.

**Comment by zicklag** [comment]
I'm not liable for issues you may have using this suggestion ( :wink: ), but another option if you are using something like S3 is just to delete the namespace from the S3 bucket. It looks like the storage format for registry is super simple. Just delete the folder named after the namespace and you're done. :smile:

**Comment by ptimar56** [comment]
Agreed with @zicklag, it works (tried on a local registry).
